library/a-star/src/main/java/astar/AStar.java

```diff
@@ -31,6 +46,111 @@ public static void linkNeighbors(Spot[][] grid) {
         }
     }
 
+    public static Map minmaxXY(JSONObject coords, int gridSize, int ratio){
```

Comment:
function name should be more descriptive imo, an action keyword or just a comment should suffice

Reply:
agreed, I changed it to pointArea as it depicts the area of the POI given the coordinate